src/data/AmendmentCenter.cpp

```diff
@@ -168,4 +182,18 @@ Amendment::GetAmendmentId(std::string_view name)
     return ripple::sha512Half(ripple::Slice(name.data(), name.size()));
 }
 
+std::optional<std::vector<ripple::uint256>>
+AmendmentCenter::fetchAmendmentsList(boost::asio::yield_context yield, uint32_t seq) const
+{
+    // the amendments should always be present on the ledger
+    auto const& amendments = backend_->fetchLedgerObject(ripple::keylet::amendments().key, seq, yield);
+    ASSERT(amendments.has_value(), "Amendments ledger object must be present in the database");
```

Comment:
Probably better to throw an exception here to return `Internal error` to a user instead of crashing Clio if connection to db lost.